docs/deployment.md

```diff
@@ -0,0 +1,75 @@
+Production deployment guide
+=====================
+
+Requirements
+-------------------
+
+- php >=7.2
+- node >=8 (Only for creation front-end assets)
+- yarn (Only for creation front-end assets)
+
+Install (with build archive)
+-------------------
```

Comment:
Are these instructions used when building an actual production release? We usually perform these steps in an ansible role in Stepup-deploy as far as I know. Or are these instructions for non-Ansible using users?

Reply:
No, also for ansible users. Now they need to do less :)